extras/container/heketi-start.sh

```diff
@@ -22,6 +29,60 @@ fail() {
     exit 1
 }
 
+archive_db() {
+    # create an archive file for the previous copy of the db
+    # before the server starts up
+    mkdir -p "${HEKETI_DB_ARCHIVE_PATH}"
+
+    # only archive the db if the db content is valid
+    if ! "$HEKETI_BIN" db export --dbfile "${HEKETI_PATH}/heketi.db" --jsonfile - >/dev/null ; then
+        error "Unable to export db. DB contents may not be valid"
+        return 1
+    fi
+
+    mapfile -t afiles < <(find "${HEKETI_DB_ARCHIVE_PATH}" \
+        -maxdepth 1 -name 'heketi.db-archive*.gz' | sort)
```

Comment:
Nitpick: could be `heketi.db-archive-*.gz`

Reply:
done